Vue InstantSearch/query-suggestions/src/App.vue

```diff
@@ -13,55 +13,56 @@
     <div class="container">
       <ais-instant-search
         :search-client="searchClient"
-        index-name="instant_search_demo_query_suggestions"
+        index-name="instant_search"
       >
-        <ais-configure :hitsPerPage="5" />
-        <ais-autocomplete>
-          <template slot-scope="{ currentRefinement, indices, refine }">
-            <vue-autosuggest
-              :suggestions="indicesToSuggestions(indices)"
-              @selected="onSelect"
-              :input-props="{
-                style: 'width: 100%',
-                onInputChange: refine,
-                placeholder: 'Search here…',
-              }"
+        <ais-index index-name="instant_search_demo_query_suggestions">
```

Comment:
having both indices means that both will always be queried, even though only one at a time changes.